src/Testcontainers.CosmosDb/CosmosDbContainerWaitStrategy.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Testcontainers.CosmosDb;
+
+public sealed class CosmosDbContainerWaitStrategy : IWaitUntil
+{
+    private const string ReadyProbeUrl = "https://localhost/_explorer/index.html";
+        
+    public async Task<bool> UntilAsync(IContainer container)
+    {
+        try
+        {
+            using var httpClient = ((CosmosDbContainer)container).HttpClient;
+
+            var result = await httpClient.GetAsync(ReadyProbeUrl);
+                
+            return result.IsSuccessStatusCode;
+        }
+        catch (Exception)
+        {
+            return false;
+        }
+    }
+}
```

Comment:
Thanks for the PR. Can you please add the class to the builder as we do in other [modules](https://github.com/testcontainers/testcontainers-dotnet/blob/1c80fe80432873a625b2cde7ce52e99308d68b4e/src/Testcontainers.MySql/MySqlBuilder.cs#L128-L149) too?